sdk/mobile-center-push/build.gradle

```diff
@@ -0,0 +1,10 @@
+//
```

Comment:
We cleaned that in updates actually. Reason is that old Android Studio was showing build.gradle as editor title so it was confusing. Now it's showing the module name in editor tab title so we don't need this comment anymore.